### PR TITLE
nightshade + web bump to 534

### PR DESCRIPTION
**Pull request by kennethgillen**
Bump versions, usage text, requirements bump

### CODE REVIEW

File: requirements.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 - name: openmicroscopy.omero-web
   src:  https://github.com/openmicroscopy/ansible-role-omero-web.git
-  version: 1.0.0-m1
```
Comment (sbesson) [inline requirements.yml]: I assume this requirement bump was also used for https://github.com/openmicroscopy/prod-playbooks/pull/3?

Comment (kennethgillen) [reply requirements.yml]: Yes sir. It was noticed on this system, and it looks like worked-around on ome-demoserver via https://github.com/kennethgillen/prod-playbooks/blob/c6455280cbebb20f84d1c8392ce78fb48c3e6785/ome-demoserver.yml#L161

Comment (kennethgillen) [reply requirements.yml]: So the upgrade to ome-demoserver was uneventful, but the change to the Nightshade ecosystem caused a broken omero-web. This was the reason.

**Comment by kennethgillen** [comment]
@sbesson - I'm not sure what `ansible-lint` is complaining about here:

```
(ansible)LS27172:prod-playbooks kenny$ ansible-lint ome-dundeeomero.yml
Couldn't parse task at Unknown (no action detected in task. This often indicates a misspelled module name, or incorrect module path.)
{ '__line__': 75,
  'become': True,
  'name': 'NGINX - enable service and start on boot',
  'systemd': { '__line__': 77, 'enabled': True, 'name': 'nginx'}}

```

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1606](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1606/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1606/consoleText) for more details.
Possible conflicts:
  - PR #3 kennethgillen 'OME OMERO demo server 5.3.4 upgrade'
    - .travis.yml
  - Upstream changes
    - .travis.yml

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1607](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1607/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1607/consoleText) for more details.
Possible conflicts:
  - PR #3 kennethgillen 'OME OMERO demo server 5.3.4 upgrade'
    - .travis.yml
  - Upstream changes
    - .travis.yml

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1608](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1608/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1608/consoleText) for more details.
Possible conflicts:
  - PR #3 kennethgillen 'OME OMERO demo server 5.3.4 upgrade'
    - .travis.yml
  - Upstream changes
    - .travis.yml

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1609](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1609/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1609/consoleText) for more details.
Possible conflicts:
  - PR #3 kennethgillen 'OME OMERO demo server 5.3.4 upgrade'
    - .travis.yml
  - Upstream changes
    - .travis.yml

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1610](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1610/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1610/consoleText) for more details.
Possible conflicts:
  - PR #3 kennethgillen 'OME OMERO demo server 5.3.4 upgrade'
    - .travis.yml
  - Upstream changes
    - .travis.yml

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1611](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1611/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1611/consoleText) for more details.
Possible conflicts:
  - PR #3 kennethgillen 'OME OMERO demo server 5.3.4 upgrade'
    - .travis.yml
  - Upstream changes
    - .travis.yml

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1612](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1612/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1612/consoleText) for more details.
Possible conflicts:
  - PR #3 kennethgillen 'OME OMERO demo server 5.3.4 upgrade'
    - .travis.yml
  - Upstream changes
    - .travis.yml

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1613](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1613/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1613/consoleText) for more details.
Possible conflicts:
  - PR #3 kennethgillen 'OME OMERO demo server 5.3.4 upgrade'
    - .travis.yml
  - Upstream changes
    - .travis.yml

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1614](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1614/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1614/consoleText) for more details.
Possible conflicts:
  - PR #3 kennethgillen 'OME OMERO demo server 5.3.4 upgrade'
    - .travis.yml
  - Upstream changes
    - .travis.yml

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1615](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1615/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1615/consoleText) for more details.
Possible conflicts:
  - PR #3 kennethgillen 'OME OMERO demo server 5.3.4 upgrade'
    - .travis.yml
  - Upstream changes
    - .travis.yml

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1616](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1616/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1616/consoleText) for more details.
Possible conflicts:
  - PR #3 kennethgillen 'OME OMERO demo server 5.3.4 upgrade'
    - .travis.yml
  - Upstream changes
    - .travis.yml

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1617](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1617/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1617/consoleText) for more details.
Possible conflicts:
  - PR #3 kennethgillen 'OME OMERO demo server 5.3.4 upgrade'
    - .travis.yml
  - Upstream changes
    - .travis.yml

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1618](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1618/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1618/consoleText) for more details.
Possible conflicts:
  - PR #3 kennethgillen 'OME OMERO demo server 5.3.4 upgrade'
    - .travis.yml
  - Upstream changes
    - .travis.yml

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1619](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1619/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1619/consoleText) for more details.
Possible conflicts:
  - PR #3 kennethgillen 'OME OMERO demo server 5.3.4 upgrade'
    - .travis.yml
  - Upstream changes
    - .travis.yml

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1620](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1620/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1620/consoleText) for more details.
Possible conflicts:
  - PR #3 kennethgillen 'OME OMERO demo server 5.3.4 upgrade'
    - .travis.yml
  - Upstream changes
    - .travis.yml

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1621](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1621/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1621/consoleText) for more details.
Possible conflicts:
  - PR #3 kennethgillen 'OME OMERO demo server 5.3.4 upgrade'
    - .travis.yml
  - Upstream changes
    - .travis.yml

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1622](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1622/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1622/consoleText) for more details.
Possible conflicts:
  - PR #3 kennethgillen 'OME OMERO demo server 5.3.4 upgrade'
    - .travis.yml
  - Upstream changes
    - .travis.yml

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1623](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1623/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1623/consoleText) for more details.
Possible conflicts:
  - PR #3 kennethgillen 'OME OMERO demo server 5.3.4 upgrade'
    - .travis.yml
  - Upstream changes
    - .travis.yml

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1624](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1624/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1624/consoleText) for more details.
Possible conflicts:
  - PR #3 kennethgillen 'OME OMERO demo server 5.3.4 upgrade'
    - .travis.yml
  - Upstream changes
    - .travis.yml

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1625](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1625/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1625/consoleText) for more details.
Possible conflicts:
  - PR #3 kennethgillen 'OME OMERO demo server 5.3.4 upgrade'
    - .travis.yml
  - Upstream changes
    - .travis.yml

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1626](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1626/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1626/consoleText) for more details.
Possible conflicts:
  - PR #3 kennethgillen 'OME OMERO demo server 5.3.4 upgrade'
    - .travis.yml
  - Upstream changes
    - .travis.yml

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1627](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1627/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1627/consoleText) for more details.
Possible conflicts:
  - PR #3 kennethgillen 'OME OMERO demo server 5.3.4 upgrade'
    - .travis.yml
  - Upstream changes
    - .travis.yml

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1628](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1628/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1628/consoleText) for more details.
Possible conflicts:
  - PR #3 kennethgillen 'OME OMERO demo server 5.3.4 upgrade'
    - .travis.yml
  - Upstream changes
    - .travis.yml

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1629](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1629/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1629/consoleText) for more details.
Possible conflicts:
  - PR #3 kennethgillen 'OME OMERO demo server 5.3.4 upgrade'
    - .travis.yml
  - Upstream changes
    - .travis.yml

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1630](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1630/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1630/consoleText) for more details.
Possible conflicts:
  - PR #3 kennethgillen 'OME OMERO demo server 5.3.4 upgrade'
    - .travis.yml
  - Upstream changes
    - .travis.yml

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1631](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1631/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1631/consoleText) for more details.
Possible conflicts:
  - PR #3 kennethgillen 'OME OMERO demo server 5.3.4 upgrade'
    - .travis.yml
  - Upstream changes
    - .travis.yml

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1632](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1632/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1632/consoleText) for more details.
Possible conflicts:
  - PR #3 kennethgillen 'OME OMERO demo server 5.3.4 upgrade'
    - .travis.yml
  - Upstream changes
    - .travis.yml

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1633](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1633/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1633/consoleText) for more details.
Possible conflicts:
  - PR #3 kennethgillen 'OME OMERO demo server 5.3.4 upgrade'
    - .travis.yml
  - Upstream changes
    - .travis.yml

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1634](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1634/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1634/consoleText) for more details.
Possible conflicts:
  - PR #3 kennethgillen 'OME OMERO demo server 5.3.4 upgrade'
    - .travis.yml
  - Upstream changes
    - .travis.yml

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1635](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1635/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1635/consoleText) for more details.
Possible conflicts:
  - PR #3 kennethgillen 'OME OMERO demo server 5.3.4 upgrade'
    - .travis.yml
  - Upstream changes
    - .travis.yml

--conflicts

**Comment by sbesson** [comment]
This playbook is modified in #7, should we unify both PRs into a single one?

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1648](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1648/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1648/consoleText) for more details.
Possible conflicts:
  - Upstream changes
    - .travis.yml

--conflicts

**Comment by sbesson** [comment]
Closing this as this conflicts. Instead we should be able to update #7 which includes the state of the current production server.

**Pull request closed without merging**